crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -240,17 +241,19 @@ mod tests {
         while let Some((spend, rest)) = a.next(all_spends) {
             all_spends = rest;
             // process the spend
-            let [parent_id, puzzle, amount, solution, _spend_level_extra] =
-                extract_n::<5>(&a, spend, ErrorCode::InvalidCondition).expect("extract_n");
-
+            let destructure_list!(parent_id, puzzle, amount, solution) =
+                <match_list!(BytesImpl<32>, Program, u64, Program)>::from_clvm(&a, spend)
+                    .expect("parsing CLVM");
```

Comment:
are you sure this has the same behavior? if `spend` is a list that has an extra element, will that still match this list?